### PR TITLE
reference: update TiKV metrics docs

### PR DESCRIPTION
**Pull request by lonng**
Signed-off-by: Lonng <heng@lonng.org>

### What is changed, added or deleted? 

The TiKV metrics documents about `raft store CPU` is stale and this PR fix this.

### CODE REVIEW

File: v3.0/reference/key-monitoring-metrics/overview-dashboard.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ aliases: ['/docs-cn/op-guide/dashboard-overview-info/']
 - coprocessor pending requests：正常情况监控为 0 或者数量很少
 - coprocessor executor count：不同类型的查询操作数量
 - coprocessor request duration：TiKV 中查询消耗的时间
-- raft store CPU：raftstore 线程的 CPU 使用率，目前为单线程，超过 80% 说明使用率很高
+- raft store CPU：raftstore 线程的 CPU 使用率，线程数量默认为 2 (通过 `raftstore.store-pool-size` 配置)，如果单个线程使用率超过 80% 说明使用率很高
```
Comment (dcalvin) [inline v3.0/reference/key-monitoring-metrics/overview-dashboard.md]:
```suggestion
- raft store CPU：raftstore 线程的 CPU 使用率，线程数量默认为 2 (通过 `raftstore.store-pool-size` 配置)，如果单个线程使用率超过 80%，说明使用率很高
```

File: dev/reference/key-monitoring-metrics/overview-dashboard.md [hunk 2]
```diff
@@ -68,7 +68,7 @@ category: reference
 - coprocessor pending requests：正常情况监控为 0 或者数量很少
 - coprocessor executor count：不同类型的查询操作数量
 - coprocessor request duration：TiKV 中查询消耗的时间
-- raft store CPU：raftstore 线程的 CPU 使用率，目前为单线程，超过 80% 说明使用率很高
+- raft store CPU：raftstore 线程的 CPU 使用率，线程数量默认为 2 (通过 `raftstore.store-pool-size` 配置)，如果单个线程使用率超过 80% 说明使用率很高
```
Comment (dcalvin) [inline dev/reference/key-monitoring-metrics/overview-dashboard.md]:
```suggestion
- raft store CPU：raftstore 线程的 CPU 使用率，线程数量默认为 2 (通过 `raftstore.store-pool-size` 配置)。如果单个线程使用率超过 80%，说明使用率很高
```

File: v3.0/reference/key-monitoring-metrics/overview-dashboard.md [hunk 3]
```diff
@@ -69,7 +69,7 @@ aliases: ['/docs-cn/op-guide/dashboard-overview-info/']
 - coprocessor pending requests：正常情况监控为 0 或者数量很少
 - coprocessor executor count：不同类型的查询操作数量
 - coprocessor request duration：TiKV 中查询消耗的时间
-- raft store CPU：raftstore 线程的 CPU 使用率，目前为单线程，超过 80% 说明使用率很高
+- raft store CPU：raftstore 线程的 CPU 使用率，线程数量默认为 2 (通过 `raftstore.store-pool-size` 配置)，如果单个线程使用率超过 80%，说明使用率很高
```
Comment (dcalvin) [inline v3.0/reference/key-monitoring-metrics/overview-dashboard.md]:
```suggestion
- raft store CPU：raftstore 线程的 CPU 使用率，线程数量默认为 2 (通过 `raftstore.store-pool-size` 配置)。如果单个线程使用率超过 80%，说明使用率很高
```

**Comment by lilin90** [comment]
@hicqu PTAL

**Pull request merged**